components/well/index.css

```
@@ -11,19 +11,18 @@ governing permissions and limitations under the License.
 */
 
 .spectrum-Well {
-  --spectrum-well-padding: var(--spectrum-global-dimension-size-200);
-  --spectrum-well-border-width: var(--spectrum-alias-border-size-thin);
-  --spectrum-well-margin-top: var(--spectrum-global-dimension-size-50);
-  --spectrum-well-min-width: var(--spectrum-global-dimension-size-3000);
+  --spectrum-well-border-width: var(--spectrum-border-width-100);
 }
 
 .spectrum-Well {
   text-align: start;
   display: block;
   min-inline-size: var(--spectrum-well-min-width);
-  padding: var(--spectrum-well-padding);
-  margin-block-start: var(--spectrum-well-margin-top);
-  border-width: var(--spectrum-well-border-width);
+  padding: var(--mod-well-padding, var(--spectrum-well-padding));
+  margin-block-start: var(--mod-well-margin-top, var(--spectrum-well-margin-top));
+  border-width: var(--mod-well-border-width, var(--spectrum-well-border-width));
   border-style: solid;
-  border-radius: var(--spectrum-alias-border-radius-regular);
+  border-radius: var(--mod-well-border-radius, var(--spectrum-well-border-radius));
+  background-color: var(--spectrum-gray-800);
```

this is missing the opacity, 

```suggestion
  background-color: rgba(var(--spectrum-gray-800-rgb), 0.02);
```

---

could add a mod for the background-color too